.github/workflows/main.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           - 27017:27017
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [18.x, 20.x]
     steps:
     - uses: actions/checkout@v2
```

Comment:
While you're here:
- Update actions to v3 (all of them are v3 right now)
- Update lint and coverage node version to 20.x.
- Update mongo to 5.